__tests__/app.test.js

```diff
@@ -29,14 +41,39 @@ describe('/api/topics', () => {
         })
     })
 })
-describe('/api', () => {
+describe('/api/articles/:article_id', () => {
     describe('GET', () => {
-        test('GET 200: responds with a json containing all the available endpoints that can be used by the user', () => {
+        test('GET 200: responds with an article object with the specified id', () => {
             return request(app)
-            .get('/api')
+            .get('/api/articles/1')
             .expect(200)
             .then(({body}) => {
-                expect(body.endpoints).toEqual(endpoints)
+                expect(body.article).toMatchObject({
+                    article_id: expect.any(Number),
```

Comment:
Would you expect this article_id to be any number? You should be more specific here